sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp

```diff
@@ -22,10 +23,10 @@
 #include <memory>
 #include <mutex>
 #include <set>
+#include <sstream>
```

Comment:
not used within the header, is it?

Move it to the cpp if it is only used there.